### PR TITLE
Add 'context' and 'info' commands to augtool; reorganize help a little

### PR DESCRIPTION
**Pull request by lutter**

### CODE REVIEW

File: src/augrun.c [hunk 1]
```diff
@@ -1116,6 +1148,49 @@ static const struct command_def cmd_load_def = {
...
+        if (entry != cmd->aug->modpathz) {
+            fprintf(cmd->out, ":");
+        }
+        fprintf(cmd->out, "%s", entry);
+    }
+    fprintf(cmd->out, "\n");
+
+    {
+
+    }
```
Comment (domcleal) [inline src/augrun.c]: Unused block?

Comment (lutter) [inline src/augrun.c]: Urgh .. yes, I wanted to print more stuff but ran out of steam

**Comment by lutter** [comment]
I am not entirely sure how useful the `info` command is in its current incarnation .. maybe leave that out for now ?

**Comment by domcleal** [comment]
It's not very useful, but if new functionality is added then it's a reasonable base to extend. It could include the path context, version of libfa etc?

**Comment by lutter** [comment]
I added context and the number of files currently loaded .. unless you can think of anything else to throw in there, I'll merge that tomorrow.

**Pull request merged**